### DISCUSSION TITLE
What means type "Literature"? Literature being read? Literature for sale?

### DISCUSSION THREAD

**Discussion by Strompf** [code4recovery/12-step-meeting-list#1229] [Q&A] [answered]
Thank you for this wonderful plugin! Maybe a silly question, but what does the type "Literature" stands for?

1. A meeting where literature is being read?
2. A meeting where literature is for sale?

Do you have a source for that? I've searched quite a bit online for this, but I couldn't find it anywhere.

Thank already!
Regards, Jeroen

BTW: For now, I assumed option (2): It seems more important to me that folks can select meetings where they can buy certain literature, than to know whether or not some literature is being read at a certain meeting

**Comment by joshreisner** [comment, marked as answer]
Hi Jeroen, I think almost all the types are subject to a certain amount of interpretation locally. Open and Closed are the only ones that have definitions included. My sense is for the most part people use definition 1) where Literature, other than the Big Book, is being read or discussed. 

Looking back at [the history](https://github.com/code4recovery/12-step-meeting-list/commit/45ddb84d82e4d65acb2dcc089dcb6d63d7f40107#diff-9ec26a19ef0cbada8cdaeaf19e42bfb308f15ef75fe7bd7bef41ea003078f603R51), this was added in 2015 before any of the other literature types like Big Book, 12 & 12, Living Sober, etc., so I suspect it was originally considered a catch-all, and may still be used by some sites as that if they don't want to specify individual titles.

**Reply by Strompf** [reply]
Thank you! I checked with some local meetings, and it seemed that they too, tended to go with your interpretation

**Comment by kiyote33** [comment]
Hi again, Sorry I didn't notice your earlier question concerning the meeting type "Literature". Our base specification doesn't really specify the exact meaning to be associated with the literature type. My personal interpretation is that it represents meetings where GSO's information pamphlets are used as the basis for discussion at the designated meeting.  Note that you could clarify the meeting desc with a code override in a child theme functions.php file, such as:

`if (function_exists('tsml_custom_descriptions')) {
    tsml_custom_descriptions([
        'LIT' => 'Literature available for sale!', or
        'LIT' => 'AA literature discussed'
    ]);
}`

Hope this helps...

**Reply by Strompf** [reply]
Thank you for the suggestion :)
